### ISSUE TITLE
no anonymous "community" blog items, please

### ISSUE THREAD

**Issue by dckc** [rchain/marketing#18]
Most items have real bylines, but some go under an anonymous "community" signature.
for example:
https://blog.rchain.coop/rchain-debrief-93-rcon3/

Please fix.

cc @Viraculous

see also https://github.com/rchain/bounties/issues/896

**Comment by bjornagain** [comment]
If this request is for a change to the blog, I don't think it belongs here. This repo is for rchain.coop.

**Comment by dckc** [comment]
really? It's called "marketing" and it has elearning and branding and other stuff. The README says:

> Welcome to the RChain Marketing Repo
> This is a public resource for RChain Marketing assets and applications.

**Comment by bjornagain** [comment]
yep, really. The blog is not currently located in this repo, and as far as I know nobody who works on the blog is looking at these issues.

**Comment by Viraculous** [comment]
@dckc Thanks, will get that fixed.
